packages/attribute-registry-process/open-api/attribute-registry-service-spec.yml

```diff
@@ -0,0 +1,590 @@
+openapi: 3.0.3
+info:
+  title: attribute registry Process Micro Service
+  description: Offers operations to retrieve attribute registry data
+  version: '{{version}}'
+  contact:
+    name: API Support
+    url: 'http://www.example.com/support'
+    email: support@example.com
+  termsOfService: 'http://swagger.io/terms/'
+  x-api-id: an x-api-id
+  x-summary: an x-summary
+servers:
+  - url: '/attribute-registry-process/{{version}}'
+    description: This service is the attribute registry process
+security:
+  - bearerAuth: []
+tags:
+  - name: attribute-registry
+    description: Implements attribute-registry process
+    externalDocs:
+      description: Find out more
+      url: 'http://swagger.io'
+  - name: health
+    description: Verify service status
+    externalDocs:
+      description: Find out more
+      url: 'http://swagger.io'
+paths:
+  '/certifiedAttributes':
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+    post:
+      summary: Creates certified attribute
+      tags:
+        - attribute
+      operationId: createCertifiedAttribute
+      responses:
+        '200':
+          description: Certified attribute created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Attribute'
+        '400':
+          description: Bad Request
+          content:
+            application/problem+json:
```

Comment:
This is not well supported by `zodios` in our service we're changing every `problem+json` into a more standard `json`

Reply:
Thanks for the heads up, we will change it to `json` as you suggested!